src/AddTodoForm.js

```diff
@@ -1,34 +1,31 @@
 import React, { useState } from "react";
 
-function AddTodoForm({ onAddTodo }) {
+function AddTodoForm(props) {
   // Create a new state variable named todoTitle with an update function named setTodoTitle
   const [todoTitle, setTodoTitle] = useState("");
```

Comment:
Since this state is no longer being used you can delete this line as well as `, { useState }` on line 1

Reply:
okay i deleted it